src/lib/race.ts

```diff
@@ -151,9 +153,11 @@ export async function race(
       store.dispatch(actions.data.resetFilters());
     },
     async changeOptions(newOptions: Partial<Options>) {
+      const validOptions = validateOptions(newOptions);
```

Comment:
shall we rename this variable to something like `newValidOptions`?

There is already a variable with the same name in the outer scope. I prefer not to shadow variables this way because it is prone to problems.

Reply:
Done